docs/controller.md

```diff
@@ -0,0 +1,93 @@
+## Controller - general information
```

Comment:
Let's start the page with `# Controller` and then on a newline let's add `## General information`.